### PR TITLE
Comments! with a magical robot guide easter egg.

### PR DESCRIPTION
**Pull request by jackbackes**

### CODE REVIEW

**Comment by jackbackes** [comment]
image

File: server/app/routes/api/v1/comments/index.js [hunk 1]
```diff
@@ -41,6 +45,55 @@ router.post( '/:id/annotation', ( req, res, next ) => {
     .then( ()=> next()).catch(next);
 
 } )
+
+router.get('/:commentId/tags', (req, res, next) => {
+  console.log(req.params.id, req.params.commentId);
+  Comment.findById(req.params.commentId)
+         .then( comment => {
+      console.log(comment);
```
Comment (jdhang) [inline server/app/routes/api/v1/comments/index.js]: chop chop 🔪

File: browser/js/components/Comment/renderComment.js [hunk 2]
```diff
@@ -47,14 +47,18 @@ export default class RenderComment extends Component {
...
     this.renderComment = this.renderComment.bind(this);
     this.handleSubmitDialog = this.handleSubmitDialog.bind(this);
     this.renderMarkdown = this.renderMarkdown.bind(this);
     this.renderScore = this.renderScore.bind(this);
-    this.removeItem = this.removeItem.bind(this);
+    this.removeItem = this.props.removeItem;
+    this.updateContents = this.props.updateContents;
+    this.renderDialogHandler = this.renderDialogHandler.bind(this);
+    this.tagMethods = this.props.tagMethods;
+    this.handleSlider = this.handleSlider.bind(this);
```
Comment (jancodes) [inline browser/js/components/Comment/renderComment.js:61]: You could use es6 to reduce the lines taken. E.g.

const { removeItem, updateContents, tagMethods, handleSlider } = this.props 😄

File: browser/js/components/Comment/index.js [hunk 3]
```diff
@@ -63,17 +65,36 @@ let defaultContents = {
   markdown: "blah blah blah"
```
Comment (jancodes) [inline browser/js/components/Comment/index.js:65]: ❓

**Comment by jancodes** [comment]
🐑 it

**Pull request merged**